ansible/tasks/stickypedia.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: "create stickypedia user"
+  user:
+    name: "stickypedia"
+    shell: "usr/sbin/nologin"
+    home: "/var/www/stickypedia"
+    system: true
+
+block:
+  - name: "clone git with Stickypedia content"
+    git:
+      repo: {{ stickypedia.git_content }}
+      dest: "/var/www/stickypedia/stickypedia.{{ canonical_hostname }}"
+      version: "master"
+    diff: false
+
+  - name: "install prerequisites for gollum"
```

Comment:
This needs to run as `root`, take it to above the block

Reply:
fixed, thanks